docs/Setup-Dev.md

```diff
@@ -0,0 +1,27 @@
+#Software to install:
+-[Visual Studio](https://visualstudio.microsoft.com/it/thank-you-downloading-visual-studio/?sku=Community&rel=16)
+-[Visual Studio Code](https://code.visualstudio.com/)
+-Client Git([Fork](https://git-fork.com/) o [GitExtensions](http://gitextensions.github.io/))
+-[NodeJS last LTS](https://nodejs.org/it/download/)
+-[MongoDB Server](https://www.mongodb.com/) 
+-[MongoDB Compass](https://www.mongodb.com/) (optional)
+-[Postman](https://www.getpostman.com/downloads/) (optional)
+
+#Configuration
+##Visual Studio's port setting the same as Postman
+On Postman import the collections: open the folder of rawCMS->docs->RawCMS.postman_collection.json.
+After that, open any collection and copy the port number from the link.
```

Comment:
at the end of the line, before the dot `.`, add between brackets the expected port number (e.g. `(should be 28436)`)

Reply:
remove `e.g.` part. The final result inside brackets should be:
```
(should be 28436)
```